data/src/main/java/com/safetyheads/data/network/retrofit/RetrofitInterceptor.kt

```diff
@@ -0,0 +1,14 @@
+package com.safetyheads.data.network.retrofit
+
+import okhttp3.Interceptor
+import okhttp3.Response
+
+class RetrofitInterceptor(private val appLogger: AppLogger) : Interceptor {
+    override fun intercept(chain: Interceptor.Chain): Response {
```

Comment:
do we need it? as well as AppLogger? There is a library for that already https://stackoverflow.com/a/33256827